commons-mongo/src/main/scala/com/avsystem/commons/mongo/core/ops/Updating.scala

```diff
@@ -3,11 +3,14 @@ package mongo.core.ops
 
 import com.avsystem.commons.mongo.BsonRef
 import com.avsystem.commons.serialization.GenCodec
+import com.mongodb.client.model.Updates
 import org.bson.conversions.Bson
 
 object Updating extends LowPrioUpdating {
   implicit def bsonUpdating(bson: Bson): BsonUpdating = new BsonUpdating(bson)
   implicit def bsonRefIterableUpdating[E: GenCodec, C[T] <: Iterable[T]](bsonRef: BsonRef[C[E]]): BsonRefIterableUpdating[E, C] = {
```

Comment:
Are you fine with the fact that this will also catch `Map`s? They don't serialize to arrays so this may fail completely silently in runtime (just a query returning empty result without any error or warning).